environment-dev.yml

```diff
@@ -7,17 +7,17 @@ dependencies:
   - cmake
   - cxx-compiler
   # Host dependencies
-  - xeus-zmq >=1.0.2,<2.0
-  - nlohmann_json=3.11.2
+  - xeus=5.1.0
+  - xeus-zmq>=3.0,<4.0
+  - nlohmann_json=3.11.3
   - cppzmq
-  - xtl
   - CppInterOp>=1.3.0
   - pugixml
   - cpp-argparse <3.1
   - zlib
   # Test dependencies
   - pytest
-  - jupyter_kernel_test>=0.4.3
+  - jupyter_kernel_test>=0.5,<0.6
```

Comment:
@anutosh491 Why is the newest version of jupyter_kernel_test (0.7.0) incompatible? I can look into making it compatible once I know why.

Reply:
Ahh no specific reason for this change. Whenever we update the xeus-stack, we update all the kernels to the same versions for dependencies mostly (mainly looking at xeus-python) . For now we've decided to go with this version of jupyter_kernel_test for consistency. I am not sure if 0.7.0 is incompatible but for the first release of xeus-cpp (1.0.0) let's use this version of jupyter_kernel_test as it will be used across all xeus stack !